packages/cactus-test-tooling/src/main/typescript/ws-test-server/ws-test-server.ts

```diff
@@ -0,0 +1,154 @@
+import { EventEmitter } from "events";
+
+import Docker, { Container } from "dockerode";
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+} from "@hyperledger/cactus-common";
+
+import { Containers } from "../common/containers";
+
+export const WS_IDENTITY_IMAGE_NAME = `brioux/ws-identity`;
```

Comment:
@brioux Did you also want to contribute this container image definiton? If yes please include the dockerfile in the PR and instructions in the README.md on how to build the image and then I can publish it on the official hyperledger/cactus-* repo (and then I'll ask to update this image tag here to use that official one instead).
This is just a question because I understand you may not want to actually contribute that image itself, which is also cool but I wanted to clarify.

Reply:
@petermetz no issue with this. basic instructions on building the ws-identity docker image are provided in the test-tooling readme as commented below.